runconfig/config.go

```diff
@@ -33,6 +33,8 @@ type Config struct {
 	NetworkDisabled bool
 	MacAddress      string
 	OnBuild         []string
+	SecurityOpt     []string
+	Label           []string
```

Comment:
I feel Labels would be more consistent with other fields such as ExposedPorts and Volumes.  I know env is not plural, but I think thats just because "envs" is weird.